src/Components/TaskReview/TaskReview.js

```diff
@@ -24,6 +44,38 @@ export default function TaskReview(props) {
       </div>))
   }
 
+  function renderOptions() {
```

Comment:
format was generally  easy to work with, worked with it in my branch for combined effort